verisignmdns/src/main/java/denominator/verisignmdns/VrsnContentConversionHelper.java

```diff
@@ -0,0 +1,86 @@
+package denominator.verisignmdns;
+
+import static denominator.model.ResourceRecordSets.a;
+import static denominator.model.ResourceRecordSets.aaaa;
+import static denominator.model.ResourceRecordSets.cname;
+import static denominator.model.ResourceRecordSets.txt;
+import static denominator.model.ResourceRecordSets.mx;
+import static denominator.model.ResourceRecordSets.naptr;
+import static denominator.model.ResourceRecordSets.ptr;
+import static denominator.model.ResourceRecordSets.srv;
+import static denominator.model.ResourceRecordSets.ns;
+import static denominator.model.ResourceRecordSets.ds;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import denominator.model.ResourceRecordSet;
+import denominator.verisignmdns.VrsnMdns.Record;
+
```

Comment:
rewrite this file like https://github.com/Netflix/denominator/blob/master/ultradns/src/main/java/denominator/ultradns/UltraDNSFunctions.java